cfme/services/myservice/ssui.py

```diff
@@ -290,3 +320,20 @@ class MyServiceEditTags(SSUINavigateStep):
 
     def step(self):
         self.prerequisite_view.policy_btn.item_select('Edit Tags')
+
+
+@navigator.register(MyService, 'Retire')
+class MyServiceRetire(SSUINavigateStep):
+    VIEW = RetireServiceView
+
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        wait_for(
```

Comment:
I'm on the fence if this should be required or not, but I'm leaning toward yes.

@psav recently made some updates to `SSUINavigateStep`, which now supports `wait_for_view` boolean arg.

I think you should drop this wait_for call, and expand your prerequisite to be a method, using `navigate_to(self.context['object'], 'Details', wait_for_view=True)`

Reply:
Done